tests/expr_and_series/over_test.py

```diff
@@ -88,17 +88,12 @@ def test_over_multiple(constructor: Constructor) -> None:
     if "duckdb" in str(constructor) and DUCKDB_VERSION < (1, 3):
         pytest.skip()
     df = nw.from_native(constructor(data))
-    expected = {
-        "a": ["a", "a", "b", "b", "b"],
-        "b": [1, 2, 3, 3, 5],
-        "c": [5, 4, 3, 1, 2],
-        "i": list(range(5)),
-        "c_min": [5, 4, 1, 1, 2],
-    }
     expected = {
         "a": ["a", "a", "b", "b", "b"],
         "b": [1, 2, 3, 5, 3],
         "c": [5, 4, 3, 2, 1],
+        "i": list(range(5)),
+        "c_min": [5, 4, 1, 2, 1],
     }
```

Comment:
here there were 2 expected, the latest of which didn't have the most important column we should test :D

That's what triggered this PR :)

Reply:
that's tragic 😄 🤦 😭 thanks!